### DISCUSSION TITLE
Replay Gain difference beetween apply by "track" and "playback order"

### DISCUSSION THREAD

**Discussion by Npa972** [DeaDBeeF-Player/deadbeef#2937] [Q&A]
Hi,
If i use RG "playback order" i think RG is not applied ? I thought it will be.

By "track" is ok.

So please can someone explain me the difference into these RG settings.

Thanks

![Capture d’écran_2023-04-05_16-27-47](https://user-images.githubusercontent.com/49346431/230204374-f6b1d9f5-b1e0-4a3e-9bac-14a19125c432.png)

**Comment by gorman42** [comment]
I'm a new user for DeadBeef but my experience with other players suggests me that "playback order" means using the album RG value if the following track is part of the same album and using track RG if the following track is part of a different album.

OR

Using album RG when playback order is normal, and track RG when playback order is shuffle by track.

The first option would be preferable IMHO. Curious about it too.

**Reply by Npa972** [reply]
OK thank's you i'll try to investigate these options..
